lib/tool_mount_point/wpcc.rb

```diff
@@ -0,0 +1,6 @@
+module ToolMountPoint
+  class Wpcc < Base
+    EN_ID = 'wpcc'
```

Comment:
Perhaps I am understand wrongly. If I follow this idea on this PR, then the old wpcc will be shut down and the new wpcc will be secret and ** inaccessible** to our users. Is this what you expected? By the way, there is an user story on the backlog: https://moneyadviceservice.tpondemand.com/entity/8433? Let's talk in person about it.